modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx

```diff
@@ -352,6 +341,44 @@ export function ModalAddFilter({
 						setOptions={setItems}
 					/>
 				)}
+
+				{selectedFilterBy?.businessType === 'Date' && (
+					<>
+						<Input
+							label={Liferay.Language.get('start')}
+							onChange={({target: {value}}) => {
+								setItems([
+									...items.filter(
+										(item) => item.value !== 'ge'
```

Comment:
Hi @danvb112. Does these `'ge'` and `'le'` terms change according to the language? Also, I think it's a little bit confusing when reading. Is there a way to make this more clear, like creating a variable or something else?

Reply:
Hey @alinedoleron, these `ge` and `le` are related to the filter operator we need to send to the backend, we are following this Liferay documentation https://help.liferay.com/hc/en-us/articles/360031163631-Filter-Sort-and-Search

Reply:
Got it!! Thank you very much for the explanation!